test/e2e-upgrade-tests.sh

```diff
@@ -45,9 +45,11 @@ function install_latest_release() {
     yaml="serving-core.yaml"
   fi
 
-  install_knative_serving \
-    "${url}/${yaml}" \
-    || fail_test "Knative latest release installation failed"
+  wget "${url}/${yaml}" -O "${yaml}" \
```

Comment:
I suggest using `mktemp` for the local file instead of `$yaml`, so you don't risk overwriting any local artifact. Plus the manifest will be kept in the temp dir for a while, which might be useful for debugging.

Reply:
done